templates/downloads/index.html

```diff
@@ -53,6 +53,8 @@ <h1 class="call-to-action">Download the latest version of Python</h1>
                     <h2 class="widget-title">Active Python releases</h2>
                     <p class="success-quote"><a href="https://devguide.python.org/versions/#versions">For more information visit the Python Developer's Guide</a>.</p>
 
+                    <p class="release-cycle-chart"><img src="https://hugovk-devguide.readthedocs.io/en/standalone-svg/release-cycle.svg" alt="Python release cycle"></p>
```

Comment:
For demo purposes.

When [devguide#1708](https://github.com/python/devguide/pull/1708) is merged, replace with:

```suggestion
                    <p class="release-cycle-chart"><img src="https://devguide.python.org/_static/release-cycle.svg" alt="Python release cycle"></p>
```